docroot/themes/humsci/humsci_basic/src/scss/objects/_layouts.row.scss

```
@@ -54,7 +54,8 @@
     }
 
     .field-hs-row-components,
-    .field-hs-collection-items {
+    .field-hs-collection-items,
+    h2 {
```

Make sure not to affect other possible `<h2>` within the component:

```suggestion
    > h2 {
```

---

Ahhh yess, nice catch!. Thanks!